scipy/linalg/tests/test_lapack.py

```diff
@@ -2923,3 +2924,29 @@ def test_ptsvx_NAG(d, e, b, x):
     df, ef, x_ptsvx, rcond, ferr, berr, info = ptsvx(d, e, b)
     # determine ptsvx's solution and x are the same.
     assert_array_almost_equal(x, x_ptsvx)
+
+
+@pytest.mark.parametrize('dtype', DTYPES)
+def test_pptrs_pptrf_ppsv(dtype):
+    seed(1234)
+    # Manual conversion to/from packed format is feasible here.
+    n, nrhs = 10, 4
+    a = generate_random_dtype_array([n, n], dtype=dtype)
+    b = generate_random_dtype_array([n, nrhs], dtype=dtype)
+    a = a.conj().T + a + np.eye(n, dtype=dtype) * dtype(5.)
```

Comment:
```suggestion
    a = a.conj().T + a + np.eye(n, dtype=dtype) * dtype(5.)
```
I suppose this happens to make it diagonally dominant, but that depends on the RNG gods, right? I know you're setting the seed so if it works, it works; just checking.

Reply:
For uniformly sampled random symmetric matrices of 10x10, all eigs except the largest ones are centered around zero following Wigner's semicircle law. So 5 is ~2x the minimum eigenvalue that is expected.

Reply:
Cholesky needs the matrix to be symmetric as well as positive definite, so this should be
```a = a.conj().T @ a + np.eye(n, dtype=dtype) * dtype(5.)```

Reply:
Probably I am blinded but isn't `a + a*` already hermitian and adding `5*I` makes it pos def?

Reply:
Yes.